h/api/models/document.py

```diff
@@ -26,17 +26,18 @@ class Document(Base, mixins.Timestamps):
     def __repr__(self):
         return '<Document %s>' % self.id
 
+    @property
+    def title(self):
+        meta = DocumentMeta.query.filter(DocumentMeta.document_id == self.id,
+                                         DocumentMeta.type == 'title').first()
+        if meta:
+            return meta.value
+
     @classmethod
-    def find_or_create_by_uris(cls, claimant_uri, uris,
-                               created=None, updated=None):
+    def find_by_uris(cls, claimant_uri, uris=[]):
```

Comment:
Is there any point making a distinction between `claimant_uri` and `uris` in this method? It's needed in the `find_or_create` version, but I don't think it's necessary here.